package_manifest.yaml

```diff
@@ -368,6 +368,7 @@ foreman_nonscl_packages:
     katello-certs-tools: {}
     katello-selinux: {}
     keycloak-httpd-client-install: {}
+    nodejs-react-html-parser: {}
```

Comment:
I think this needs to be moved up to the foreman_nodejs_packages group above this group.  It's whitelisted for scl, but the foreman_nonscl_packages group doesn't build against that tag.

Reply:
Also, I think this needs to be broken out into it's own PR, as it seems this is a new build time dependency for foreman-tasks